Sources/ApodiniUtils/Shell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  File.swift
+//
+//
+//  Created by Eldi Cano on 30.03.21.
+//
+
+import Foundation
+
+/// Contains predefined shell command cases
+public enum ShellCommand {
```

Comment:
As far as I remember @lukaskollmer also has some Schell execution abstraction in his Deployment Provider Kit, maybe moving this here into ApodiniUtlils and merging it with your code might be a nice idea to share it with more modules.

Reply:
@PSchmiedmayer My child process API (see [Task.swift](https://github.com/Apodini/Apodini/blob/0f9d7cb794042c8460e88f9e225b3ee0a4a0bd2f/Sources/ApodiniUtils/Task.swift)) is already in the ApodiniUtils target.

It conceptually works a bit different from the `ShellCommand` API implemented here (this one seems to assume that the command is run in zsh, whereas mine allows launching any executable as a child command), but I think it should be possible to merge the two. Especially since, at the end of the day, both are using the same underlying system API (NSTask)